code/implementation/docker/machine/src/main/java/io/cattle/platform/docker/machine/launch/WebsocketProxyLauncher.java

```diff
@@ -51,12 +51,14 @@
 
     protected void prepareConfigFile() throws IOException {
         String config = API_FILTER_PROXY_CONFIG.get();
+        String configContent;
         if (StringUtils.isBlank(config)) {
-            new File(API_FILTER_PROXY_CONFIG_FILE.get()).delete();
+            configContent = "{}";
```

Comment:
@ibuildthecloud  @alena1108 Ok, I updated the PR to not write the empty file. Instead reverted to deleting file as before
please review